src/plugins/console/public/application/components/help_popover.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPopover,
+  EuiTitle,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiSpacer,
+} from '@elastic/eui';
+import { useServicesContext } from '../contexts';
+
+interface HelpPopoverProps {
+  button: any;
+  isOpen: boolean;
+  closePopover: () => void;
+  resetTour: () => void;
+}
+
+export const HelpPopover = ({ button, isOpen, closePopover, resetTour }: HelpPopoverProps) => {
+  const { docLinks } = useServicesContext();
+
+  return (
+    <EuiPopover
+      button={button}
+      isOpen={isOpen}
+      closePopover={closePopover}
+      anchorPosition="downRight"
+      buffer={4}
+      data-test-subj="consoleHelpPopover"
+    >
+      <EuiTitle size="xs">
+        <h4>
+          {i18n.translate('console.helpPopover.title', {
+            defaultMessage: 'Elastic Console',
+          })}
+        </h4>
+      </EuiTitle>
+
+      <EuiSpacer size="s" />
+
+      <EuiText style={{ width: 300 }} color="subdued" size="s">
+        <p>
+          {i18n.translate('console.helpPopover.description', {
+            defaultMessage:
+              'Use the Console UI to call Elasticsearch APIs and view their responses. Use Query DSL syntax to search your data, manage settings, and more.',
```

Comment:
```suggestion
              'Console is an interactive UI for calling Elasticsearch and Kibana APIs and viewing their responses. Search your data, manage settings, and more, using Query DSL and REST API syntax.',
```

cause we changed it in other PR too 😄

Reply:
Yes, I missed this, thanks! Fixed in [d638d91](https://github.com/elastic/kibana/pull/191092/commits/d638d911fa7f9b0b69269e4465077f42fadc59ec)